torch/_C/__init__.pyi.in

```diff
@@ -71,15 +68,15 @@ from torch.utils._python_dispatch import TorchDispatchMode
 
 # This module is defined in torch/csrc/Module.cpp
 
-K = TypeVar("K")
-T = TypeVar("T")
-S = TypeVar("S", bound=torch.Tensor)
-P = ParamSpec("P")
-ReturnVal = TypeVar("ReturnVal", covariant=True)  # return value (always covariant)
-_T_co = TypeVar("_T_co", covariant=True)
+K = TypeVar("K")  # noqa: PYI001
+T = TypeVar("T")  # noqa: PYI001
+S = TypeVar("S", bound=torch.Tensor)  # noqa: PYI001
+P = ParamSpec("P")  # noqa: PYI001
+R = TypeVar("R", covariant=True)  # return value (always covariant)  # noqa: PYI001
+T_co = TypeVar("T_co", covariant=True)  # noqa: PYI001
```

Comment:
Since it's "standard practice" why not have all of these start with an underscore? At least for R and T_co it can't be BC because you renamed them here...

Reply:
These typevars are used in `gen_pyi.py`. The variable name without the underscore is easier to access.

https://github.com/pytorch/pytorch/blob/004dad48f760f0b2e9ec7ea7b82cc9b65cf91f93/tools/pyi/gen_pyi.py#L1118-L1132